resolwe_api/resolwe.py

```diff
@@ -493,3 +337,102 @@ def __call__(self, request):
         # if r.path_url != '/upload/':
         #     r.headers['X-SubscribeID'] = self.subscribe_id
         return request
+
+
+class ResolweQuerry(object):
+    """
+    Enable querries on data, collection and sample endpoints in Resolwe.
+
+    Each Resolwe instance (for example "re") has threee "endpoints": re.data,
+    re.collections and re.sample. Each such andpooint is an instance of
+    ResolweQuerry class. ResolweQuerry enables querries on
+    corresponding objects, for example:
+
+    re.data.get(42) # return Data object with ID 42.
+    re.sample.filter(contributor=1) # return all samples made by contributor 1
+
+    Detailed description of methods can be found in method docstrings.
+    """
+
+    def __init__(self, resolwe, type_):
+        self.resolwe = resolwe
+        self.object = type_
```

Comment:
Idea with capitalized attributes is from `boto3` (SKD for AWS). `Resource` is class on server, so they try to mimic that: https://boto3.readthedocs.io/en/latest/guide/quickstart.html#using-boto-3

    # Upload a new file
    data = open('test.jpg', 'rb')
    s3.Bucket('my-bucket').put_object(Key='test.jpg', Body=data)